newsfragments/298.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fix cross-platform compilation using ``distutils._msvccompiler.MSVCCompiler`` -- by :user:`saschanaz` and :user:`Avasam` 
```

Comment:
Oh, gosh. I'm realizing now that we can't use these news fragments. They refer to issues in this repository, which is merged into Setuptools, so adding them here will cause setuptools to render them as their own issues. I'll just delete it.

Reply:
Oh right!
Well you could still have newsfragments without issue # if it starts with `+` (for example: `+distutils298.bugfix.rst`)

Reply:
Good idea. I'll aim to do things like that in the future.